src/workerd/api/util.h

```diff
@@ -58,6 +58,7 @@ kj::Maybe<kj::ArrayPtr<const char>> readContentTypeParameter(kj::StringPtr conte
 // Assumptions:
 //   - `contentType` has a single Content-Type parameter, e.g. "boundary" or "charset".
 //   - `contentType` has a semi-colon followed by OWS before the parameter name.
+//   - parameter values do not use quoted-string encoding
```

Comment:
I think we need to spell out what this means. Perhaps include an example of a string which wouldn't be handled correctly.

Reply:
I improved the comment.